components/script/dom/performanceresourcetiming.rs

```diff
@@ -116,7 +116,7 @@ impl PerformanceResourceTiming {
             initiator_type: initiator_type,
             next_hop: next_hop,
             worker_start: 0.,
-            redirect_start: 0.,
+            redirect_start: resource_timing.redirect_start as f64,
```

Comment:
should I set this to fetch_start value instead?
> 20.2 If the value of redirectStart is not set, let it be the value of fetchStart.

Reply:
No. We want to keep that logic in the code you already modified.